js/rpg_core/Utils.js

```diff
@@ -125,3 +125,8 @@ Utils.rgbToCssColor = function(r, g, b) {
     b = Math.round(b);
     return 'rgb(' + r + ',' + g + ',' + b + ')';
 };
+
+Utils._id = 1;
+Utils.generateRuntimeId = function(){
+    return Utils._id++;
```

Comment:
Don't you think about overflow?
If the scene switches once per second, it reaches `Infinity` after 300 million years! 😂